### PR TITLE
Accordion

### PR DESCRIPTION
**Pull request by stianlik**
## 📥 Proposed changes

Basic accordion implementation.

## ☑️ Submission checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

-   [x] I have read the [CONTRIBUTING]() doc
-   [ ] Lint and unit tests pass locally with my changes
-   [ ] `yarn build` works locally with my changes
-   [ ] I have added tests that prove my fix is effective or that my feature works
-   [ ] I have added necessary documentation (if appropriate)
-   [ ] Any dependent changes have been merged and published in downstream modules

## 💬 Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did, what alternatives you considered, etc...

### CODE REVIEW

File: packages/accordion-react/example/index.html [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<html>
+    <link rel="stylesheet" href="../../accordion/accordion.scss" />
```
Comment (lfbergee) [inline packages/accordion-react/example/index.html]: Kan ikke gå uten for rot i pakken sin. Om man trenger noe utenfor sin egen pakke, så må man importere som om det var en pakke på npm.

File: packages/accordion-react/src/Accordion.tsx [hunk 2]
```diff
@@ -0,0 +1,33 @@
+import React, {ReactNode, useState} from 'react';
+import chevron from "../../accordion/chevron.svg";
```
Comment (lfbergee) [inline packages/accordion-react/src/Accordion.tsx]: Samme, må impoteres fra @fremtind/jkl-accordion

Comment (piofinn) [reply packages/accordion-react/src/Accordion.tsx]: Husk da å bruke `lerna add` for å legge til pakken som dependency, ellers vil den ikke funke som den skal

**Comment by lfbergee** [comment]
@all-contributors please add @stianlik  for code

**Pull request merged**